Classes/ios/UIViewController+MBDebugPanel.m

```diff
@@ -0,0 +1,157 @@
+//
+//  UIViewController+MBDebugPanel.m
+//  MBDebugPanel
+//
+//  Created by Ian on 3/27/15.
+//
+//
+
+#import "UIViewController+MBDebugPanel.h"
+
+#import "MBDebugPanel.h"
+
+#import <JGMethodSwizzler/JGMethodSwizzler.h>
+
+#import <objc/runtime.h>
+
+NSString *const MB_UIViewController_Utility_debugMenuItemsManagementEnabledKey = @"MB_UIViewController_Utility_debugMenuItemsManagementEnabled";
+NSString *const MB_UIViewController_Utility_managedDebugMenuItemsKey = @"MB_UIViewController_Utility_managedDebugMenuItemsKey";
```

Comment:
Fixed, at times I use an NSString when the key has other purposes but here there isn't. 

On the other hand I do use a value so that I can:
* ensure no collisions
* easily debug